### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by philippglasmacher**
Note that the navbar is different for people who are signed in and for those who are not. did not do the styling, currently the logout option is still in the dropdown - changeable, we will vote tomorrow.

### CODE REVIEW

File: app/assets/stylesheets/components/_navbar.scss [hunk 1]
```diff
@@ -10,3 +10,35 @@
...
+//   background-color: #008CBA;
+//   border: none;
+//   border-radius: 5%;
+//   color: white;
+//   padding: 15px 32px;
+//   text-align: center;
+//   text-decoration: none;
+//   display: inline-block;
+//   font-size: 16px;
+// }
```
Comment (ThibaultPaulmann) [inline app/assets/stylesheets/components/_navbar.scss:40]: comments can be removed, using bootstrap is good for responsive design

File: app/assets/stylesheets/components/_navbar.scss [hunk 2]
```diff
@@ -10,3 +10,35 @@
...
+
+// .nav-item {
+//   color: white;
+// }
```
Comment (ThibaultPaulmann) [inline app/assets/stylesheets/components/_navbar.scss:44]: remove if not used

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -10,8 +10,9 @@
     <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
   </head>
   <body>
+  <%= render "shared/navbar" %>
```
Comment (ThibaultPaulmann) [inline app/views/layouts/application.html.erb:13]: indentation :)

File: app/views/shared/_navbar.html.erb [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+          <li class="nav-item active">
+            <%= link_to "My bookings", "#", class: "nav-link" %>
+          </li>
+          <li class="nav-item">
+            <%= link_to "My listings", "#", class: "nav-link" %>
+          </li>
+          <li class="nav-item dropdown">
+            <%= image_tag "https://iconape.com/wp-content/files/mf/367366/png/367366.png", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
+            <div class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdown">
+              <%= link_to "Action", "#", class: "dropdown-item" %>
```
Comment (ThibaultPaulmann) [inline app/views/shared/_navbar.html.erb:23]: should we just have sign in/sign out functionality in the dropdown?

File: app/views/shared/_navbar.html.erb [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    <div id="logo-text"><h3><%=Vanify%></h3></div>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto">
+        <% if user_signed_in? %>
+          <li class="nav-item active">
+            <%= link_to "My bookings", "#", class: "nav-link" %>
```
Comment (ThibaultPaulmann) [inline app/views/shared/_navbar.html.erb:15]: implement the link routes already, they will be created if they aren't already

**Review by ThibaultPaulmann — COMMENTED** [review]
Very good work, just some minor tweaks and we can merge it :)

File: app/assets/stylesheets/components/_navbar.scss (hunk 1 above)
Comment (philippglasmacher) [reply app/assets/stylesheets/components/_navbar.scss:40]: done

File: app/assets/stylesheets/components/_navbar.scss (hunk 2 above)
Comment (philippglasmacher) [reply app/assets/stylesheets/components/_navbar.scss:44]: done

**Review by ThibaultPaulmann — APPROVED** [review]

**Pull request merged**